### PR TITLE
media-video/shotcut: version bump to 19.04.30

### PR DESCRIPTION
**Pull request by donahue95**
Signed-off-by: Drake Donahue <donahue95@comcast.net> — GCO/DCO approval (defined by GLEP 76 [4]),
Bug: https://bugs.gentoo.org/683684
Bug: https://bugs.gentoo.org/652260

### CODE REVIEW

**Comment by a17r** [comment]
You can edit the title of a PR, why did you create a new one?

File: media-video/shotcut/shotcut-19.04.30.ebuild [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	media-libs/libvpx
+	>=media-libs/mlt-6.10.0-r1[ffmpeg,frei0r,qt5,sdl,xml]
+	media-libs/x264
+	media-plugins/frei0r-plugins
+	media-sound/lame
+	media-video/ffmpeg
+	virtual/jack
+"
+DEPEND="${RDEPEND}
+	dev-qt/linguist-tools:5
```
Comment (a17r) [inline media-video/shotcut/shotcut-19.04.30.ebuild]: this dep would be in `BDEPEND` instead, in EAPI-7.

File: media-video/shotcut/shotcut-19.04.30.ebuild [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (a17r) [inline media-video/shotcut/shotcut-19.04.30.ebuild]: this is easily ported to EAPI-7, can you do it?

**Comment by a17r** [comment]
ping

File: media-video/shotcut/shotcut-19.04.30.ebuild (hunk 2 above)
Comment (donahue95) [reply media-video/shotcut/shotcut-19.04.30.ebuild]: no

Comment (a17r) [reply media-video/shotcut/shotcut-19.04.30.ebuild]: please understand that github is not just a fancy alternative to copy-pasting ebuilds in bugzilla. we expect the submitters to do some work and help them improve at that.

**Pull request closed without merging**